src/modules/dashboard/sagas/extensions/index.ts

```diff
@@ -0,0 +1,25 @@
+import { takeEvery } from 'redux-saga/effects';
+
+import { ActionTypes } from '~redux/index';
+
+import { colonyExtensionInstall } from './colonyExtensionInstall';
+import { colonyExtensionEnable } from './colonyExtensionEnable';
+import { colonyExtensionDeprecate } from './colonyExtensionDeprecate';
+import { colonyExtensionUninstall } from './colonyExtensionUninstall';
+import { colonyExtensionUpgrade } from './colonyExtensionUpgrade';
+import { updateWhitelist } from './updateWhitelist';
+
+export default function* colonyExtensionsSagas() {
+  yield takeEvery(ActionTypes.COLONY_EXTENSION_INSTALL, colonyExtensionInstall);
+  yield takeEvery(ActionTypes.COLONY_EXTENSION_ENABLE, colonyExtensionEnable);
+  yield takeEvery(
+    ActionTypes.COLONY_EXTENSION_DEPRECATE,
+    colonyExtensionDeprecate,
+  );
+  yield takeEvery(
+    ActionTypes.COLONY_EXTENSION_UNINSTALL,
+    colonyExtensionUninstall,
+  );
+  yield takeEvery(ActionTypes.COLONY_EXTENSION_UPGRADE, colonyExtensionUpgrade);
+  yield takeEvery(ActionTypes.WHITELIST_UPDATE, updateWhitelist);
+}
```

Comment:
Each individual `takeEvery` for each individual action, needs to be added to each own's saga.

This is so that the action _(ex: `COLONY_EXTENSION_UPGRADE`)_ is in the same file as it's used so that it will make debugging easier.

The structure should look something like this:

Saga file (example `colonyExtensionUpgrade`):
```js
function* colonyExtensionUpgrade({ ... }); // notion that there's no export for this function

export default function* colonyExtensionUpgradeSagas() { // this is what it's actually exported
  yield takeEvery(ActionTypes.COLONY_EXTENSION_UPGRADE, colonyExtensionUpgrade);
};
```
Inside the `sagas/extensions/index`:
```js
import { colonyExtensionUpgradeSaga } from './colonyExtensionUpgrade';

export default function* extensionsSagas() {
  yield all([
    call(colonyExtensionUpgradeSaga),
  ]);
}
```

Then inside `sagas/index` add the `extensionsSagas` to the `setupDashboardSagas`:
```js
import extensionsSagas from './extensions';

export default function* setupDashboardSagas() {
  yield all([
    ...
    call(extensionsSagas),
  ]);
}
```